ArduPlane/quadplane.cpp

```diff
@@ -3560,7 +3562,8 @@ bool QuadPlane::verify_vtol_land(void)
                 plane.set_next_WP(plane.mission.get_current_nav_cmd().content.location);
             } else {
                 plane.set_next_WP(plane.next_WP_loc);
-                plane.next_WP_loc.alt = ahrs.get_home().alt;
+                plane.next_WP_loc.set_alt_cm(ahrs.get_home().alt,
```

Comment:
Ideal place to use `copy_alt_from` (916bc6a06db8e83d3a32c4914ac29b2dec80eded in my repo)

... and there's several of those in this PR